src/style.css

```diff
@@ -1 +1,96 @@
-/* Styles go here */
+@font-face {
+  font-family: "Roboto";
+  src: url(./fonts/Roboto-Medium.ttf);
+  font-weight: 500px;
+}
+
+@font-face {
+  font-family: "Roboto-bold";
+  src: url(./fonts/Roboto-Bold.ttf);
+  font-weight: 800;
+}
+
+.header {
+  display: flex;
+  font-family: "Roboto", sans-serif;
+  justify-content: space-between;
+  min-width: 1024px;
+  max-width: 1200px;
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header:hover {
+  height: 60px;
+  background: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo {
+  margin: 10px 0 10px 50px;
```

Comment:
Please add padding horizontal to .header selector instead adding margins to logo/nav, it should be smth like this:
![image](https://user-images.githubusercontent.com/54936044/231001306-76a9dee7-1261-43c2-8e9e-db47865abbc3.png)